content/courses/native-onchain-development/hello-world-program.md

```diff
@@ -335,38 +333,57 @@ Now let's build and deploy our program using Solana Playground.
 
 #### 5. Invoke Program
 
-Finally, let's invoke our program from the client side. The focus of this lesson
-is to build our Solana program, so we’ve gone ahead and provided
-[the client code to invoke our “Hello, world!” program](https://github.com/Unboxed-Software/solana-hello-world-client)
-for you to download.
+In this final step, we'll invoke the "Hello, world!" Solana program from the client side. While the main focus of this lesson is on building your Solana program, we've provided [the client code](https://github.com/Unboxed-Software/solana-hello-world-client) to help you easily interact with your deployed program.
```

Comment:
Moved to https://github.com/solana-developers/hello-world-client - can you please make your updates there instead and fix these links?